provisioning/roles/environment/tasks/main.yml

```diff
@@ -336,3 +336,19 @@
   command: touch /home/vagrant/nifi-toolkit-{{ nifi_version }}-bin.zip
   args:
     creates: /home/vagrant/nifi-toolkit-{{ nifi_version }}-bin.zip
+
+- name: Ensure {{ home_dir }}/source exists
+  file:
+    path: "{{ home_dir }}/source"
+    state: directory
+    owner: "{{ ansible_user }}"
+    group: "{{ ansible_user }}"
+  when: copy_source
+
+- name: Copy in nifi_processors source
+  copy:
+    src: "{{ role_path }}/../../../nifi_processors"
+    dest: "{{ home_dir }}/source"
+    owner: "{{ ansible_user }}"
+    group: "{{ ansible_user }}"
+  when: copy_source
```

Comment:
The pom.xml file in the root of the repo needs to be copied into the source directory to compile the source code with maven.

Reply:
@rthomas320 oh okay so add a copy from the root `pom.xml` to `{{ home_dir }}/source/pom.xml` right?